distribution/smarthome/conf/smarthome.cfg

```diff
@@ -26,4 +26,5 @@ com.eclipsesource.jaxrs.swagger.config:swagger.info.version=1.0
 # Configuration of thread pool sizes
 org.eclipse.smarthome.threadpool:thingHandler=3
 org.eclipse.smarthome.threadpool:discovery=3
-org.eclipse.smarthome.threadpool:safeCall=3
\ No newline at end of file
+org.eclipse.smarthome.threadpool:safeCall=3
+org.eclipse.smarthome.threadpool:astro=30
```

Comment:
We must not add any binding specific configuration in here.
Why would the astro binding require 30 concurrent threads in any case?